test_report_generation.sh

```diff
@@ -0,0 +1,29 @@
+echo "Generating Reports"
+
+set -e
+
+echo "Creating report directory"
+cd features/test_reports
+date=$(date '+%Y_%m_%d')
+
+echo $date
+mkdir -p $date
+
+echo "cd to directory"
+cd $date
+
+echo Create reports
+behave --junit
+
+echo "Create overview report"
+junit2html ./reports/ --merge reports/report.xml
+junit2html ./reports/report.xml reports/report.html
+
+echo "Remove junk files"
+cd ./reports
+find . -name "*.xml" -type f|xargs rm -f
+
+echo Push to branch
+git add .
```

Comment:
in terms of how this process is gonna work, my understanding is we cant push directly to main so are we going to make a new branch each week to run the report? im wondering would it be better to have a separate repo to store the reports on that we can just push directly to main?

Reply:
@JasonBellONS  sounds like a good plan to me :) we can wait till Steve gets back to decide whether to do this and if so, which repo it should goto :)